.golangci.yaml

```
@@ -133,7 +133,7 @@ linters-settings:
       - pkg: sigs.k8s.io/cluster-api/cmd/clusterctl/client/config
         alias: configclient
       # CAPI Operator
-      - pkg: sigs.k8s.io/cluster-api-operator/api/v1alpha1
+      - pkg: sigs.k8s.io/cluster-api-operator/api/v1alpha2
```

I do not think we need to remove this, but rather add a new entry for new API version because you are using current API version in https://github.com/kubernetes-sigs/cluster-api-operator/pull/190/files#diff-c444f711e9191b53952edb65bfd8c644419fc7695c62611dc0fb304b4fb197d6R43 as `operatorv1alpha1`?

```suggestion
      - pkg: sigs.k8s.io/cluster-api-operator/api/v1alpha1
        alias: operatorv1alpha1
```